config/spotbugs/filter.xml

```diff
@@ -59,4 +59,8 @@
         <Class name="~software\.amazon\.smithy\.plugin\.language\.SmithySyntaxHighlighter"/>
         <Bug pattern="NP_NONNULL_RETURN_VIOLATION"/>
     </Match>
+    <Match>
+        <Class name="~software\.amazon\.smithy\.plugin\.language\.SmithySyntaxHighlighter"/>
+        <Bug pattern="EI_EXPOSE_REP"/>
```

Comment:
is this needed? sounds like it is the issue you fixed in ModelDependencies.java

Reply:
Yes, this added suppression covers the exposed representation issues in https://github.com/awslabs/smithy-intellij/blob/main/src/main/java/software/amazon/smithy/plugin/language/SmithySyntaxHighlighter.java

Reply:
oh didn't realize this is for specific class. it is not worth fixing the warning in that class?

Reply:
since this is following https://intellij-sdk-docs-cn.github.io/intellij/sdk/docs/tutorials/custom_language_support/syntax_highlighter_and_color_settings_page.html , sounds good.